src/components/ButtonWithMenu.js

```diff
@@ -0,0 +1,111 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import Button from './Button';
+import ButtonWithDropdown from './ButtonWithDropdown';
+import PopoverMenu from './PopoverMenu';
+
+const propTypes = {
+    /** Text to display for the menu header */
+    menuHeaderText: PropTypes.string,
+
+    /** Callback to execute when the main button is pressed */
+    onPress: PropTypes.func,
```

Comment:
Is there any reason we wouldn't want this to be required?

Reply:
Ah, good catch this should be required